EIPS/eip-5753.md

```diff
@@ -0,0 +1,247 @@
+---
+eip: 5753
+title: Lockable Extension for EIP-721
+description: Interface for locking and unlocking EIP-721 tokens.
+author: Filipp Makarov (@filmakarov)
+discussions-to: https://ethereum-magicians.org/t/lockable-nfts-extension/8800
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-05
+requires: 165, 721
+---
+
+## Abstract
+This standard is an extension of [EIP-721](./eip-721.md). It introduces lockable NFTs. The locked asset can be used in any way except by selling and/or transferring it. The owner or operator can lock the token. When a token is locked, the unlocker address (an EOA or a contract) is set. Only the unlocker is able to `unlock` the token. 
```

Comment:
A high level question: One could choose to support single unlocker or multiple unlockers. Have you considered this possibility?